docs/metalk8s-ui.rst

```diff
@@ -0,0 +1,49 @@
+Metalk8s UI
+===========
+
+
+
+
+.. todo::
+
+  The following test has been realized on vagrant only - One should list
+  all requirements for reaching the UI using a operationnal cluster.
+
+
+Gather required information
+----------------------------
+
+Run all following commands as ``root`` on the bootstrap node:
+
+.. code-block:: shell
+
+      $ ip a show eth1
```

Comment:
It may not always be `eth1`, no?

Reply:
From @MonPote : for now it's only on eth1.

Reply:
Hum, in fact the naming depends on the OS (on my laptop `eth1` is called `enp0s31f6`).

Most of the time the first/default interface is `eth1` because this was the historical naming, and it may be still true for VMs that we have control over, but in "real life" I'm not sure we can be sure that an interface called `eth1` will exists.

Reply:
It's Vagrant-only - not even valid for CI. We need to change this message (as I already said in a previous review)

Reply:
I added more precision:
```
Get the workload plane IP of the bootstrap node.
On vagrant you can obtain it with:
.. code-block:: shell

      $ ip a show eth1
```